cmd/ak/cmd/vars/vars.go

```diff
@@ -0,0 +1,96 @@
+package vars
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+)
+
+// Flags shared by all the subcommands.
+var env, project, conn string
+
+var varsCmd = common.StandardCommand(&cobra.Command{
+	Use:     "vars",
+	Short:   "Variable subcommands",
+	Aliases: []string{"var", "vs", "v"},
```

Comment:
In accordance with #229:

```suggestion
	Use:   "var",
	Short: "Connection variable subcommands: set, get, delete",
```